gtsam/linear/LinearSolver.h

```diff
@@ -0,0 +1,70 @@
+/* ----------------------------------------------------------------------------
+
+ * GTSAM Copyright 2010, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+ * See LICENSE for the license information
+
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file    LinearSolver.h
+ * @brief   Common Interface for Linear Solvers
+ * @author  Fan Jiang
```

Comment:
add Gerry, Mandy

Reply:
done, for those files which I made reasonable contributions